jobs/haproxy/spec

```diff
@@ -114,3 +118,28 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
+  keepalived.vip:
+    description: VIP that will be given to master
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
+    default: check_haproxy
+  keepalived.healthcheck_command:
+    description: check that will be performed
+    default: killall -0 haproxy
+  keepalived.interface:
+    description: interface keepalived will use to mount the VIP
+    default: eth0
+  keepalived.masterip:
+    description: IP of masternode
+  keepalived.vrrp_instance_name:
+    description: name of the VRRP instance
+  keepalived.master_priority:
+    description: master weight
+    default: 101
+  keepalived.slave_priority:
+    description: slave weight
+    default: 100
+  keepalived.lvs_id:
+    description : Keepalived process identifier
```

Comment:
it will erase the index number though, so if you have this manifest : 
```yaml
jobs:
- instances: 2
  name: public_haproxy_z1
  networks:
  - name: cf_haproxy1
  persistent_disk: 0
  resource_pool: small_z1
  templates:
  - name: haproxy
    release: cf-haproxy-keepalived
- instances: 2
  name: public_haproxy_z2
  networks:
  - name: cf_haproxy2
  persistent_disk: 0
  resource_pool: small_z1
  templates:
  - name: haproxy
    release: cf-haproxy-keepalived
```
you will have : 
vm_name/index | spec.name
--- | --- 
public_haproxy_z1/0 | public_haproxy_z1
public_haproxy_z1/1 | public_haproxy_z1
public_haproxy_z2/0 | public_haproxy_z2
public_haproxy_z2/1 | public_haproxy_z2